vue/src/Tag/TagEdit.vue

```diff
@@ -693,13 +697,9 @@ export default defineComponent({
 
         this.isDirty = false;
 
-        const idTag = response.value;
-
         TagsStore.reload(this.idContainer, this.idContainerVersion).then(() => {
-          MatomoUrl.updateHash({
-            ...MatomoUrl.hashParsed.value,
-            idTag,
-          });
+          // Go back to the list of tags
+          this.cancel();
```

Comment:
Non-blocking question: is there a reason we can't this.cancel() without even calling the reload?